### ISSUE TITLE
[4.0] Atum repaint sidebar (RFC)

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#33573]
Posting this as a new issue now that the repainted atem has been merged.

I am really struggling to focus on the white text on a blue background in the sidebar. After some experimentation the problem only seems to bother me when the text is bold.

I would like to propose removing 
```
.sidebar-wrapper .item-level-1>a {
    font-weight: 700; 
}
```

### Before - Bold
![before](https://user-images.githubusercontent.com/1296369/117278398-a4ca2b80-ae58-11eb-8120-6a3726777830.gif)

### After - regular
![after](https://user-images.githubusercontent.com/1296369/117278380-9f6ce100-ae58-11eb-911f-e3921d5aa427.gif)

**Comment by PhilETaylor** [comment]
https://github.com/joomla/joomla-cms/pull/33582

**Comment by wilsonge** [comment]
@ciar4n do you have any opinions here

related: https://github.com/joomla/joomla-cms/pull/33582

**Comment by ciar4n** [comment]
Well my personal prefrernece is bold as it offers some distinction between parent and child items in the menu. Also imo just fits better within the design.

Regardless, Im not presious about it. If my opinion on this is in the minority, I can live with the change :)

**Comment by PhilETaylor** [comment]
Pleased to say on the MacBook Pro laptop that I have been working on today remotely. The bold colour is a lot less in your face than it is on the iMac. Dramatically so. I guess there is a lot of variation based on the machine and the browser used.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33573">issues.joomla.org/tracker/joomla-cms/33573</a>.</sub>

**Comment by Fedik** [comment]
I guess it is fixed after #33582 was merged?

> The bold colour is a lot less in your face than it is on the iMac. Dramatically so.

Do not compare "remote" desktop.

**Comment by PhilETaylor** [comment]
> remote

Remote ... as in a coffee shop on laptop and not at a halogen lit office iMac on desk - not remote control :-)
